### DISCUSSION TITLE
Tutorial projects using Oceananigans

### DISCUSSION THREAD

**Discussion by johnryantaylor** [CliMA/Oceananigans.jl#3249] [Show and tell]
Hi everyone! 
We recently made a series of tutorial projects for the FDSE summer school using Oceananigans, [here](https://github.com/johnryantaylor/FDSE).  Everyone is very welcome to use them.  I wrote them to (hopefully) be accessible for people who are brand new to Julia and Oceananigans.  Each project has a README with a project description plus example scripts that are ready to run and modify.  Hopefully these will be useful for others, e.g. as a small teaching resource or as a very gentle onramp for people learning Julia and Oceananigans.  As always, comments and suggestions are very welcome!
John

**Comment by glwagner** [comment]
That's great! Thanks for advertising that @johnryantaylor !
